mentoring/answer.py

```diff
@@ -70,43 +174,39 @@ def student_input(self):
 
         return student_input
 
-    def mentoring_view(self, context=None):
-        if not self.read_only:
-            html = render_template('templates/html/answer_editable.html', {
-                'self': self,
-            })
-        else:
-            html = render_template('templates/html/answer_read_only.html', {
-                'self': self,
-            })
-        
-        fragment = Fragment(html)
-        fragment.add_css_url(self.runtime.local_resource_url(self.xblock_container, 'public/css/answer.css'))
-        fragment.add_javascript_url(self.runtime.local_resource_url(self.xblock_container,
-                                                                    'public/js/answer.js'))
-        fragment.initialize_js('AnswerBlock')
-        return fragment
+    def fallback_view(self, view_name, context=None):
+        context = context or {}
+        context['self'] = self
+        html = loader.render_template('templates/html/answer_editable.html', context)
```

Comment:
OK, I will swap out `fallback_view` for separate `mentoring_view` and `student_view` methods.